PersonApi/V1/Factories/ResponseFactory.cs

```diff
@@ -38,11 +39,38 @@ public PersonResponseObject ToResponse(Person domain)
                 DateOfBirth = FormatDateOfBirth(domain.DateOfBirth),
                 PersonTypes = domain.PersonTypes,
                 Links = _apiLinkGenerator?.GenerateLinksForPerson(domain),
-                Tenures = domain.Tenures?.Select(x => ToResponse(x)).ToList(),
+                //Tenures = domain.Tenures?.Select(x => ToResponse(x)).ToList(),
+                Tenures = SortTentures(domain.Tenures),
                 Reason = domain.Reason
             };
         }
 
+        private static List<TenureResponseObject> SortTentures(IEnumerable<Tenure> tenures)
+        {
+            if (tenures == null) return null;
+
+            // 1. Filter and order active tenures - newest to oldest
```

Comment:
I think you might need to check for `Secure` vs `Unsecure` as well?

Reply:
Thanks Michael. I re-read the requirements, and yes, you're correct. I've updated the method, and it's also now much cleaner.